ui/styles/colors.js

```diff
@@ -1 +1,2 @@
 export const lightColor = "rgba(255, 255, 255, 0.7)";
+export const seaGreenColor = "#396679"
```

Comment:
Can we please call this color "deepSpaceSparkle"?

Reply:
Yeah sure I will rename it.